### PR TITLE
2024-07-26-adapt-documentation-interlis-import-export-etc

### PR DESCRIPTION
**Pull request by sjib**
Updating documentation to tww 2024.0

- [x] new version of toolbar image added
- [x] new INTERLIS button images
- [x] some small corrections and precisions
- [x] correct image names
- [x] update default_values

### CODE REVIEW

**Comment by sjib** [comment]
```
tww-docs.admin-guide [fr] - Creating download job
tww-docs.extension-guide [de] - Creating download job
tww-docs.extension-guide [de] - 404, not_found: Resource with id 'o:teksi:p:tww-docs:r:extension-guide' does not have any content
tww-docs.extension-guide [fr] - 404, not_found: Resource with id 'o:teksi:p:tww-docs:r:extension-guide' does not have any content
tww-docs.admin-guide [fr] - Done
tww-docs.features-guide [de] - Done
Aborted
tww-docs.admin-guide [de] - Done
make: *** [Makefile:96: transifex_pull] Error 1
```

@cymed I now get this error when wanting to update the documentation.

**Comment by sjib** [comment]
@urskaufmann Does this look ok like that?

**Comment by sjib** [comment]
Is this paragraph still ok ? https://teksi.github.io/wastewater/user-guide/How-To/index.html#setting-default-values
..
2. Connect all occurences of your field name to tww_sys.get_default_values(field_name). Per default, the fields fk_provider and fk_dataowner are already connected to the function tww_sys.get_default_values(field_name). An example SQL script how to insert the default value for all occurences of a field name can be found [here](https://github.com/teksi/wastewater/blob/main/datamodel/changelogs/0001/14_default_values.sql))

**Comment by sjib** [comment]
Should solve https://github.com/teksi/wastewater/issues/339

File: docs/en/admin-guide/interlis-io/index.rst [hunk 1]
```diff
@@ -100,18 +105,43 @@ Then, confirm the dialog and choose where to save the `.xtf` file.
...
+You can check the protocol window and the text logs for further details.
+
+There are four different log files:
+
+Export step | Description | Log File
+-------- | -------- | --------
+Export schema   | Creation of export schema with ili2db and the selected INTERLIS model file | *.ili2pg-schemaimport.log
+Data export intermediatary schema   | Export of data from tww_od to the new intermediary schema, translation from English to German | *.tww2ili-export.log
+Data export to xtf | Data export from intermediary schema to INTERLIS Transfer file (xtf)   | *.ili2pg-export-MODELNAME.log
+INTERLIS validation   | Validation of the created xtf file(s) with ilivalidator   | *.ilivalidator-DSS_2020_1_LV95.log
```
Comment (urskaufmann) [inline docs/en/admin-guide/interlis-io/index.rst:129]: should it not be: ... *.ilivalidator-MODELNAME.log

File: docs/en/installation-guide/images/tww_project_qm_language_files.jpg
Comment (urskaufmann) [inline docs/en/installation-guide/images/tww_project_qm_language_files.jpg:1]: renaming is not the solution in this case!

File: docs/en/user-guide/How-To/images/configuration_default_values.png
Comment (urskaufmann) [inline docs/en/user-guide/How-To/images/configuration_default_values.png:1]: not a good picture. The first thing I see is red error text!

**Comment by urskaufmann** [comment]
Renaming the pictures is not a solution (QGIS 2.14 etc )!
I work on the editing chapter with actual pics.

File: docs/en/installation-guide/images/tww_project_qm_language_files.jpg
Comment (sjib) [reply docs/en/installation-guide/images/tww_project_qm_language_files.jpg:1]: I also updated the image.

File: docs/en/user-guide/How-To/images/configuration_default_values.png
Comment (sjib) [reply docs/en/user-guide/How-To/images/configuration_default_values.png:1]: In one way I agree - But we have to show how to solve this (-:

File: docs/en/user-guide/How-To/index.rst [hunk 2]
```diff
@@ -219,16 +219,29 @@ The Feature Attributes window for the overflow characteristic appears:
...
+
+.. figure:: images/default_values_fieldname_selection.png
+
+4. Select organisation that should be used (the corresponding obj_id will be added in the table itself).
+
+.. figure:: images/default_values_organisation_obj_id_selection.png
+
+
+
+Connect all occurences of your field name to ``tww_sys.get_default_values(field_name)``.
```
[the remark this answers is not in the document]
Comment (sjib) [reply docs/en/user-guide/How-To/index.rst:238]: Ok, I will move that.

**Comment by sjib** [comment]
> Renaming the pictures is not a solution (QGIS 2.14 etc )! I work on the editing chapter with actual pics. #345

This will create error during integration because we have a keyword check for qgep implemented.

File: docs/en/user-guide/How-To/index.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (sjib) [reply docs/en/user-guide/How-To/index.rst:258]: New chapter db-production-readiness created https://github.com/teksi/wastewater/pull/335/files#diff-3807b0164dadd03144b84143ce5ad8a09fe4f04b2abe8ec4dc4019885c686ead

**Review by urskaufmann — COMMENTED** [review]
Thanks, it will not be the last version. I have not tested several things in the HowTo Chapter. The parts hydraulic modeling have soon to be rewriten...  next doc-version

**Comment by sjib** [comment]
@urskaufmann Ich hoffe ich habe jetzt überall deine Version der Bilder drin im Kapitel editing. Danke wenn du nochmals kurz draufschaust und dann als Reviewer Accept wählst falls ok.

**Review by urskaufmann — APPROVED** [review]
ok, thanks

**Pull request merged**